src/main/java/crazypants/enderzoo/item/ItemGuardiansBow.java

```diff
@@ -110,15 +110,15 @@ public void onPlayerStoppedUsing(ItemStack stack, World worldIn, EntityLivingBas
         
         entityarrow.setDamage(entityarrow.getDamage() + damageBonus + 20);
         
-        worldIn.spawnEntityInWorld(entityarrow);
+        worldIn.spawnEntity(entityarrow);
       }
 
       worldIn.playSound((EntityPlayer) null, entityplayer.posX, entityplayer.posY, entityplayer.posZ, SoundEvents.ENTITY_ARROW_SHOOT, SoundCategory.NEUTRAL,
           1.0F, 1.0F / (itemRand.nextFloat() * 0.4F + 1.2F) + drawRatio * 0.5F);
 
       if (!arrowIsInfinite) {
-        --itemstack.stackSize;
-        if (itemstack.stackSize == 0) {
+        itemstack.shrink(1);
+        if (itemstack.getCount() == 0) {
           entityplayer.inventory.deleteStack(itemstack);
```

Comment:
have you checked with the normal bow's code that this should be called that way? The stack now isEmpty() and there's no way to find out what is was, so I don't see what that method could do with it.